tutorials/Mouse_Events/README.md

```diff
@@ -415,3 +415,236 @@ fun main() = singleWindowApplication {
     }
 }
 ```
+
+### New experimental onClick handlers (only for Desktop-JVM platform)
+`Modifier.onClick` provides independent callbacks for clicks, double clicks, long clicks. It handles clicks originated only from pointer events and accessibility `click` event is not handled out of a box.
+
+Each `onClick` can be configured to target specific pointer events (using `matcher: PointerMatcher` and `keyboardModifiers: PointerKeyboardModifiers.() -> Boolean`). `matcher` can be specified to choose what mouse button should trigger a click. `keyboardModifiers` allows for filtering pointer events which have specified keyboardModifiers pressed.   
+   
+Multiple `onClick` modifiers can be chained to handle different clicks with different conditions (matcher and keyboard modifiers).   
+Unlike `clickable`, `onClick` doesn't have a default `Modifier.indication` and `Modifier.semantics`, which need to be added separately if necessary.   
+The most generic (with the least number of conditions) `onClick` handlers should be declared before other to ensure correct events propagation.
+
+```kotlin
+import androidx.compose.animation.AnimatedContent
+import androidx.compose.animation.ExperimentalAnimationApi
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.LocalIndication
+import androidx.compose.foundation.PointerMatcher
+import androidx.compose.foundation.background
+import androidx.compose.foundation.indication
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.onClick
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.input.pointer.PointerButton
+import androidx.compose.ui.input.pointer.isAltPressed
+import androidx.compose.ui.input.pointer.isShiftPressed
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.singleWindowApplication
+
+@OptIn(ExperimentalFoundationApi::class, ExperimentalAnimationApi::class)
+fun main() {
+    singleWindowApplication {
+        Column {
+            var topBoxText by remember { mutableStateOf("Click me") }
```

Comment:
Better to write more info which action should the user perform. Something like: "Click me with LMB or Shift+LMB" (and add `, textAlign = TextAlign.Center` to the text)